code/modules/projectiles/guns/energy/special.dm

```diff
@@ -96,27 +96,6 @@
 	turret_is_lethal = 0
 	turret_sprite_set = "net"
 
-/obj/item/gun/energy/beegun
-	name = "\improper NanoTrasen Portable Apiary"
-	desc = "An experimental firearm that converts energy into bees, for purely botanical purposes."
-	icon = 'icons/obj/guns/gyrorifle.dmi'
-	icon_state = "gyrorifle"
-	item_state = "gyrorifle"
```

Comment:
From what I can see at a quick glance, this icon / dmi file is only used for the beegun.
If all the code referencing it is removed, the icon/dmi should be removed aswell.